test/unified_malloc_framework/umf_pools/disjoint_pool.cpp

```diff
@@ -72,6 +73,15 @@ TEST_F(test, freeErrorPropagation) {
 INSTANTIATE_TEST_SUITE_P(disjointPoolTests, umfPoolTest,
                          ::testing::Values(makePool));
 
+INSTANTIATE_TEST_SUITE_P(
+    disjointPoolTests, umfMemTest,
+    ::testing::Values(std::make_tuple(
+        [] {
+            return umf_test::makePoolWithOOMProvider<usm::DisjointPool>(
+                4, poolConfig());
```

Comment:
If this 4 is the same 4 as in `config.Capacity` it would be better to make it a constant or just call poolConfig().Capacity here.

Reply:
Similarly for '2'. Is it calculated from the config somehow or is it just an arbitrary number?

Reply:
Done. `2` was an arbitrary number. I changed it to `poolConfig().Capacity / 2` though because I thought that might be a better idea if for example the `Capacity` in config would change.